spark/v3.3/spark/src/test/java/org/apache/iceberg/spark/sql/TestAggregatePushDown.java

```diff
@@ -595,12 +593,12 @@ public void testNaN() {
       explainContainsPushDownAggregates = true;
     }
 
-    Assert.assertTrue(
-        "explain should contain the pushed down aggregates", explainContainsPushDownAggregates);
+    Assert.assertFalse(
```

Comment:
I initially wanted to push down `NaN`, but I think it's better not to. I changed to not push down if there are `NaN`

Reply:
Yeah, not pushing when there are NaN values seems like a good idea. Spark has different behavior than Trino in this case.